gapis/api/vulkan/api/memory.api

```diff
@@ -211,7 +211,7 @@ cmd VkResult vkFlushMappedMemoryRanges(
       }
     }
 
-    if !(flushRange.memory in DeviceMemories) { vkErrorInvalidDeviceMemory(flushRange.memory) }
+    if !(flushRange.memory in DeviceMemories) { vkErrorInvalidDeviceMemory(flushRange.memory) } else {
```

Comment:
Is there a reason for not bothering indenting this block of code?

Side comment: I don't like inline if statements, however I won't push new styles if it's the standard for these .api source files.

Reply:
So the reason that we have been doing this is because we can't early-return in the API language.

We use this pattern so that we don't add incredible amounts of nesting.
What I REALLY want to say is 
```
if !(flush.memory in DeviceMemories) { return; }
```